### ISSUE TITLE
yebashInit() is called in tests

### ISSUE THREAD

**Issue by Mrokkk** [szborows/yebash#18]

**Comment by szborows** [comment]
due to `__constructor__`?

**Comment by Mrokkk** [comment]
Yes. We have to find a way to disable it. By the way, I find it out, when I started configuring Travis - it gave me that "Could not open history file" exception ;)
